### DISCUSSION TITLE
New meshio version

### DISCUSSION THREAD

**Discussion by davidcortesortuno** [pyvista/pyvista#1725] [Q&A]
Hi, in August `meshio` was updated to versions `5.x` https://github.com/nschloe/meshio/releases. Pyvista requires versions `<5` but looking at the code it seems `meshio` is not used much in the code. Can the installation files be updated to support the newest `meshio` versions?

**Comment by akaszynski** [comment]
Most likely it will work, but we should validate that.  Would you mind checking this out on your environment and then submitting a PR if it works?
